x-pack/plugins/timelines/server/search_strategy/timeline/index.ts

```diff
@@ -84,6 +84,7 @@ const timelineSearchStrategy = <T extends TimelineFactoryQueryTypes>({
   logger: Logger;
 }) => {
   const dsl = queryFactory.buildDsl(request);
+  options.retrieveResults = true;
```

Comment:
should do the same in the timelineSessionsSearchStrategy below

Reply:
nit: you can just put one `options.retrieveResults = true` on line 44 😄